nano-layout.el

```diff
@@ -16,9 +16,11 @@
 ;; along with this program. If not, see <http://www.gnu.org/licenses/>.
 ;; ---------------------------------------------------------------------
 
+(require 'disp-table)
+
 (setq default-frame-alist
       (append (list
-	       '(font . "Roboto Mono:style=Light:size=14")
+	       '(font . "Roboto Mono:style=Medium:size=18")
```

Comment:
Can you put back default size/weight?

Reply:
Yes. Done. Sorry about that. Careless of me.